### ISSUE TITLE
AWS Allocate and associate Elastic IP fails, "parameters are required together: device_id, private_ip_address"

### ISSUE THREAD

**Issue by jcklein** [StreisandEffect/streisand#1614]
Dear all,
I'm trying to set up a wireguard server on an AWS host with Streisand, my local machine is a Mac.

Setting up Streisand worked without a problem. However, I haven't been able to resolve the script failing at the "Allocate and associate Elastic IP" step. I can allocate an elastic IP from the AWS console, so in principle it seems the service works.

The error message I get is attached below. I've tried using streisand's default settings, and I've also tried creating a new VPC and subnet. I can reach the EC2 instance via SSH after this error message appears, so it seems the creation of the instance works fine.
Many thanks for your advice. I'm new to Streisand and AWS, so it is entirely possible I'm making a trivial mistake. 
Best,
Johannes

TASK [genesis-amazon : Create CloudWatch alarm to auto-recover instance] *******
changed: [localhost]

TASK [genesis-amazon : Wait until the server has finished booting and OpenSSH is accepting connections] ***
ok: [localhost]

TASK [genesis-amazon : Allocate and associate Elastic IP] **********************
**fatal: [localhost]: FAILED! => {"changed": false, "msg": "parameters are required together: device_id, private_ip_address"}**

PLAY RECAP *********************************************************************
localhost                  : ok=20   changed=11   unreachable=0    failed=1    skipped=1    rescued=0    ignored=0   

(venv) $ 

#### *Log output from Ansible or other relevant services (link to Gist for longer output):*
streisand hasn't generated any logs
##### *Target Cloud Provider:*
Amazon EC2
##### *Operating System of target host:*
Ubuntu
##### *Operating System of client:* 
MacOS 10.14.6

**Comment by nickgnazzo** [comment]
Got the same error, looks like this is a bug in ansible 2.8.0 (see [Github issue here](https://github.com/ansible/ansible/issues/58472)).

It looks like it was fixed in ansible 2.8.3, and is being [backported](https://github.com/ansible/ansible/pull/55194#issuecomment-512729474) to 2.8.0, but it doesn't seem like the backported version is available yet on PyPI (it still shows the version as being [released in May](https://pypi.org/project/ansible/#history)). I tried re-downloading 2.8.0 with pip and that didn't seem to have the bugfix either.

As a temporary workaround, you can update your Streisand virtualenv ansible version to 2.8.3 which fixes the bug. This worked for me locally when I tried against my AWS account:

```
cd /path/to/streisand
source venv/bin/activate
pip install ansible==2.8.3
./streisand
```

Seemed to finish running the provisioning scripts without any errors, but I didn't test with every Streisand service enabled. My guess is 2.8.3 wouldn't break any of the existing ansible playbooks but could be worth another test.

**Comment by jcklein** [comment]
Dear Nick,
Thank you, that fixed it. I wouldn't have been able to fix that, and I'm amazed at how quickly I got a response on here. Time to start playing with Barbra!
Cheers,
Johannes
